src/test/java/com/opendxl/databus/util/ConsumerHelper.java

```diff
@@ -120,7 +121,10 @@ public ConsumerHelper commit() {
     }
 
     public ConsumerHelper close() {
-        consumer.close();
+        try {
+            consumer.close();
+        } catch (IOException e) {
+        }
```

Comment:
Add log  or stactrace same as previous comment.

Reply:
It is just for testing